openai-connector-demo/src_hd/com/openai/connector/demo/GetAssistants/GetAssistants.xhtml

```diff
@@ -0,0 +1,35 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+  xmlns:f="http://xmlns.jcp.org/jsf/core"
+  xmlns:h="http://xmlns.jcp.org/jsf/html"
+  xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+  xmlns:ic="http://ivyteam.ch/jsf/component"
+  xmlns:p="http://primefaces.org/ui"
+  xmlns:pe="http://primefaces.org/ui/extensions">
+<h:body>
+  <ui:composition template="/layouts/frame-10-full-width.xhtml">
+    <ui:define name="title">GetAssistants</ui:define>
+    <ui:define name="content">
+      <h3>Get list of Assistants</h3>
+      <p:dataTable value="#{data.assistants.data}" var="assistant">
```

Comment:
looks as if hold the complete technical 'AssistantResponse' on the dialog, while we only make use of the 'assistants.data'  ... so I'd rather simplify by just storing this sub-attribute on process.data.